hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -241,13 +241,18 @@ static const struct usage_map xpadneo_usage_maps[] = {
 	USAGE_IGN(0x9001E), /* copy of RS */
 	USAGE_IGN(0xC0082), /* copy of Select button */
 
-	/* XBE2: Disable unused "TRIGGER_HAPPY" buttons */
-	USAGE_IGN(0x90012),
+	/* XBE2: Disable extra features until proper support is implemented */
```

Comment:
If you want to get this merged in this state, you should add this to the docs, too: Put some information that the extra buttons and functions are not supported yet, and that only profile 0 has been tested.

Reply:
I tried to find the user-facing controller-specific sections, but I'm not sure there is one? I can add a new section to the readme, or put these notes next to the descriptor. Which one would you prefer?

Reply:
I think it should be in the `README.md`, somewhere near the feature description.